modules/reference/pages/node-properties.adoc

```diff
@@ -116,14 +116,14 @@ Flag to enable developer mode, which skips most of the checks performed at start
 
 === empty_seed_starts_cluster
 
-Controls how a new cluster is formed. This property must have the same value in all brokers in a cluster.
+Controls how a new cluster is formed. All brokers in a cluster must have the same value.
 
-If `true`, an empty <<seed_servers,seed_servers>> list denotes that this broker should form a cluster. At most, one broker in the cluster should be configured with an empty seed_servers list. If no such configured broker exists, or if configured to be `false`, then all brokers denoted by the seed_servers list must be identical in their configurations, and those brokers form the initial cluster.
-
-TIP: This is set to `true` by default for backward compatibility, but Redpanda recommends setting `empty_seed_starts_cluster` to `false`.
+<<seed_servers,See how the empty_seed_starts_cluster setting works with the seed_servers setting>> to form a cluster.
```

Comment:
```suggestion
<<seed_servers,See how the `empty_seed_starts_cluster` setting works with the `seed_servers setting`>> to form a cluster.
```

Reply:
Thanks for all the help on this commit! 
I think you meant 
> `seed_servers` setting

instead of 
> `seed_servers` setting`